test/defaultbrowsercontext.jest.js

```diff
@@ -18,25 +18,32 @@
 const fs = require('fs');
 const utils = require('./utils');
 const {makeUserDataDir, removeUserDataDir} = utils;
-const {FFOX, MAC, CHROMIUM, WEBKIT, WIN, USES_HOOKS} = utils.testOptions(browserType);
+const {FFOX, MAC, CHROMIUM, WEBKIT, WIN, USES_HOOKS} = testOptions;
 
 describe('launchPersistentContext()', function() {
-  async function launch(state, options = {}) {
-    state.userDataDir = await makeUserDataDir();
-    state.context = await state.browserType.launchPersistentContext(state.userDataDir, {...state.defaultBrowserOptions, ...options});
-    state.page = state.context.pages()[0];
-    return state;
-  }
-  async function close(state) {
-    await state.context.close();
-    await removeUserDataDir(state.userDataDir);
-    delete state.page;
-    delete state.context;
-    delete state.userDataDir;
-  }
+  let userDataDir;
```

Comment:
```js
registerFixture('launch', async ({browserType, defaultBrowserOptions}) => {
    const userDataDir = await makeUserDataDir();
    const context = await browserType.launchPersistentContext(userDataDir, {...defaultBrowserOptions, ...options});
    await test({ userDataDir, context, page: context.pages()[0] });
    await context.close();
    await removeUserDataDir(userDataDir);
});
```

Reply:
I can see your launch is a function with parameters, you can have it being a function too!

Reply:
made it a fixture